etc/utils.js

```diff
@@ -176,4 +176,20 @@ if (m > 6) {
 	}
 }
 
-exports.sponsorsScope = sponsorsScope;
\ No newline at end of file
+exports.sponsorsScope = sponsorsScope;
+
+exports.makeUserPaid = function(req, res, nextSteps) {
```

Comment:
Yeah, there was a typo. It's being used in credits.js. I'll pull it out of utils, I guess it's unlikely to be used anywhere else